README.md

```diff
@@ -80,15 +80,20 @@ be stable.
 
 ## License
 
-The code samples are licensed under MIT license
+The code samples are licensed under the MIT license. See
+[LICENSE](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt)
+for more information.
 
-## Known issues or limitations
+## Known Issues or Limitations
 
 ### On Windows Platform
-1. If you are using Visual Studio 2019, Visual Studio 2019 version 16.4.0 or newer is required.
-2. To build samples on Windows, the required Windows SDK is ver. 10.0.17763.0. 
-3. Now you should be able to build the code sample.
-4. FPGA samples support Windows through FPGA-emulator.
-5. If you encounter a compilation error like below when building a sample program, one reason is that the directory path of the sample is too long; the work around is to move the sample to a directory like "c:\temp\sample_name".
-  * Error MSB6003 The specified task executable "dpcpp.exe" could not be run .......
+- If you are using Visual Studio 2019, Visual Studio 2019 version 16.4.0 or newer is required.
+- Windows SDK version 10.0.17763.0 is required to build samples on Windows.
+- FPGA samples support Windows through the FPGA-emulator.
```

Comment:
Please change this line to say:
"FPGA code sample Windows support is limited to the FPGA emulator and optimization reports. See any FPGA code sample for details."